certora/specs/StakingRouterInvariants.spec

```diff
@@ -21,6 +22,7 @@ invariant StakingModuleIndexIsIdMinus1(uint256 moduleId)
 
 invariant StakingModuleId(uint256 moduleId)
     getStakingModuleIdById(moduleId) == moduleId
+    //same comment - is rule sanity passing on the insertion function ?
```

Comment:
Again, this is only valid for modules that are registered.
I can maybe change the invariant that says ("or moduleId > count => getId == 0")

Reply:
that's a much better approach, invariants should not cause vacuity. One should assume that it is also safe to do requireinvariant. in this case it is not